api/ImageData.json

```
@@ -148,6 +148,68 @@
                 "deprecated": false
               }
             }
+          },
+          "pixelFormat_option": {
+            "__compat": {
+              "spec_url": "https://html.spec.whatwg.org/multipage/canvas.html#canvascolortype",
```

Let's add a description (while we're here, let's also add one for `colorSpace_option` and `settings_parameter` above):

```suggestion
              "description": "`pixelFormat` option",
              "spec_url": "https://html.spec.whatwg.org/multipage/canvas.html#canvascolortype",
```